sdk/daml-script/runner/src/main/scala/com/daml/lf/engine/script/v1/ScriptF.scala

```diff
@@ -251,7 +257,17 @@ object ScriptF {
         optR <- client.queryContractId(parties, tplId, cid)
         optR <- Converter.toFuture(
           if (asDisclosure)
-            Right(optR.map(c => SValue.SText(c.blob.toHexString)))
+            Right(
+              optR.map(c =>
+                makePair(
+                  record(
+                    StablePackagesV2.TemplateTypeRep,
+                    ("getTemplateTypeRep", STypeRep(TTyCon(c.templateId))),
+                  ),
```

Comment:
I'd also be surprised if something for this doesn't exist in converter.
But if it doesn't, maybe we should move it there

Reply:
I saw that `fromTemplateTypeRep` in the converter only accepted a proto identifier, and I saw that somewhere else in the converter, that code was being duplicated whenever a a non-proto identifier was needed. So I thought maybe the converter was only about converting proto to svalue. Sounds like not. I've added an overload to `fromTemplateTypeRep` in the converter and used it everywhere now.